src/scripts.js

```diff
@@ -65,49 +65,44 @@ function instantiateClasses(userData, ingredData, recipeData) {
   currentUser = new User(userData[0]);
```

Comment:
We need to generate a random user on page load instead of user at 0. To do this, lets try something like: `new User(userData[Math.floor( Math.random() * userData.length ])`. Math.floor should generate a random number in our users array.